docs/guides/specifying-dependencies.rst

```diff
@@ -16,18 +16,11 @@ To use the requirements file, create and place the requirements file in the root
 
     docs/requirements.txt
 
-Using the YAML configuration file
----------------------------------
-
-The recommended approach for specifying a pip requirements file is to use a ``readthedocs.yml`` file. 
-
-The file's path should be relative to documentation root.
+Using a configuration file
+--------------------------
 
-.. code-block:: yaml
-
-    requirements_file: requirements.txt
-
-See :doc:`/yaml-config` for setting up the .yml file
+The recommended approach for specifying a pip requirements file is to use a :doc:`/config-file/index` file, 
+see :ref:`config-file/v2:Requirements file`.
 
 Using the project admin dashboard
 ---------------------------------
```

Comment:
We may suggest the user that it's preferable to use the config file here, I suppose.

Reply:
We just mention that above